### PR TITLE
Clarify `ember test --path` docs.

### LINKED ISSUE #5122: `Ember test` builds a package every time
When using `ember --filter` for different scenarios like

```shell
# package.json
"test": "ember test --f 'unit' --host='myorg.srx.com' && ember test --f 'jshint' --host='myorg.srx.com' && ember test --host='app.srx.com' -f 'app.srx.com' -e 'test'"
```

a build is built 3 times. Is it possible to cache it?

Related to : https://github.com/ember-cli/ember-cli/issues/5121

### PR DESCRIPTION
**Pull request by rwjblue**
Fixes #5122

### CODE REVIEW

File: lib/commands/test.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = Command.extend({
     { name: 'launch',      type: String,  default: false,                            description: 'A comma separated list of browsers to launch for tests.' },
     { name: 'reporter',    type: String,                            aliases: ['r'],  description: 'Test reporter to use [tap|dot|xunit]' },
     { name: 'test-page',   type: String,                                             description: 'Test page to invoke' },
-    { name: 'path',        type: String,                                             description: 'Path to a build to run tests against.' },
```
Comment (stefanpenner) [inline lib/commands/test.js]: maybe, "reuse an existing build"?

Comment (rwjblue) [inline lib/commands/test.js]: Updated

Comment (stefanpenner) [inline lib/commands/test.js]: LGTM

**Comment by rwjblue** [comment]
Node 5 CI failure is related to https://github.com/ember-cli/ember-cli/issues/5126.

**Comment by stefanpenner** [comment]
ci is broken being os npm v5 bug, rest of the CI runs are fine.

**Pull request merged**